draft-ietf-tls-tls13.md

```diff
@@ -2819,11 +2822,19 @@ MUST behave in one of three ways:
 In order to accept early data, the server MUST have accepted a
 PSK cipher suite and selected the first key offered in the
 client's "pre_shared_key" extension. In addition, it MUST verify that
-the following values are consistent with those negotiated in the
+the following values are consistent with those associated with the selected
+PSK.  For externally established PSKs, the associated values are those
+provisioned along with the key.  For PSKs established via a NewSessionTicket
+message, the associated values are those negotiated in the
 connection during which the ticket was established.
 
-- The TLS version number and cipher suite.
+- The TLS version number
+- The selected cipher suite.
 - The selected ALPN {{RFC7301}} protocol, if any.
+- The selected SNI (Section 3 of {{RFC6066}}) value, if any.
```

Comment:
I would move the text on external vs. ticket after the list so that you can use a colon.  e.g.,

```
In addition, it MUST verify that
the following values are consistent with those associated with the selected
PSK:

- ...

These requirements are a superset of those needed to perform a 1-RTT
handshake using the PSK in question.  For externally established PSKs,
the associated values are those provisioned along with the key.  For
PSKs established via a NewSessionTicket message, the associated
values are those negotiated in the connection during which the ticket
was established.
```